python/ray/serve/_private/application_state.py

```diff
@@ -773,6 +776,9 @@ def get_docs_path(self, app_name: str) -> Optional[str]:
     def get_route_prefix(self, name: str) -> Optional[str]:
         return self._application_states[name].route_prefix
 
+    def get_ingress_deployment(self, name: str) -> Optional[str]:
+        return self._application_states[name].ingress_deployment
```

Comment:
Yes that is possible, I've improved how this is handled since it is a public API. If the application name doesn't exist, the api will throw an explicit error.